pkg/channel/message_receiver_test.go

```diff
@@ -218,7 +218,7 @@ func TestMessageReceiver_ServerStart_trace_propagation(t *testing.T) {
 	server := httptest.NewServer(kncloudevents.CreateHandler(r))
 	defer server.Close()
 
-	require.NoError(t, tracing.SetupStaticPublishing(logger.Sugar(), "", tracing.AlwaysSample))
+	require.NoError(t, tracing.SetupStaticPublishing(logger.Sugar(), "localhost", tracing.AlwaysSample))
```

Comment:
I think you can remove that, it's a leftover from a previous pr probably

Reply:
@n3wscott is this change relevant to this PR?